<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGEditTextDialog</class>
 <widget class="QDialog" name="GGEditTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cmbStyles"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnItalic">
       <property name="text">
        <string notr="true">I</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtText"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GGEditTextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GGEditTextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbStyles</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GGEditTextDialog</receiver>
   <slot>applyStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnItalic</sender>
   <signal>toggled(bool)</signal>
   <receiver>GGEditTextDialog</receiver>
   <slot>toggleItalic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtText</sender>
   <signal>currentCharFormatChanged(QTextCharFormat)</signal>
   <receiver>GGEditTextDialog</receiver>
   <slot>updateStyle(QTextCharFormat)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applyStyle()</slot>
  <slot>toggleItalic()</slot>
  <slot>updateStyle(QTextCharFormat)</slot>
 </slots>
</ui>
